### DISCUSSION TITLE
MooseRevision.h is missing

### DISCUSSION THREAD

**Discussion by yangbai90** [idaholab/moose#22294] [Q&A Getting Started] [answered]
In the `moose/modules` folder, when executing `make all`, it will fail due to the missing header file, where the header file `MooseRevision.h` can not be found.

In the `moose/moose.mk` config file, at line 302, it still requires this header file:
```
moose_revision_header := $(FRAMEWORK_DIR)/include/base/MooseRevision.h
```

Copy the `MooseRevision.h` file from previous version into that folder can solve this issue.

Should we either remove the `line-302` of `moose/moose.mk` or recovery the `MooseRevision.h`?

Best,
Yang

**Comment by GiudGiud** [comment, marked as answer]
Hello

Sorry we missed this, we dont use issues for user support. 
Your python installation is missing some packages. See similar posts in the discussions forum
https://github.com/idaholab/moose/discussions/22035

Guillaume
